### ISSUE TITLE
requires slow5 when using -x hpc-mid (or maybe any profile?)

### ISSUE THREAD

**Issue by mparker2** [hasindu2008/f5c#94]
Hi @hasindu2008 

I'm trying out f5c (cpu version 0.7) for the first time and ran it with the command:

```
$F5C eventalign \
  --rna \
  -x hpc-mid \
  --scale-events \
  --signal-index \
  --summary summary.txt  \
  -r reads.fastq  \
  -b alignments.bam \
  -g transcriptome.fasta
```

I found that using `-x hpc-mid`, or in fact any of the profile presets I tried, f5c seems to require a slow5 file and gives the error:
```
[slow5_open_with::ERROR] Argument 'pathname' cannot be NULL. At src/slow5.c:254
[init_core] Error opening SLOW5 file (null)
```

It starts processing the data without error if I remove the `-x` flag.

Looking forward to using f5c :)
Matt

**Comment by hasindu2008** [comment]
Looks like I have introduced a silly bug while introducing slow5 file support. I will fix it soon sometime this week. For now, given you are using the CPU-only version, you could set the options set by -x hpc-mid  that affects the CPU manually by:
$F5C eventalign \
  --rna \
  --scale-events \
  --signal-index \
  --summary summary.txt  \
  -r reads.fastq  \
  -b alignments.bam \
  -g transcriptome.fasta \
 -t 48 --iop 48 -K 2048 -B 20M

**Comment by hasindu2008** [comment]
Thanks for reporting this. This has been fixed in the latest 0.8 version.
